app/src/main/java/com/example/drawingApp/Layer.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.drawingApp
+
+import android.graphics.Bitmap
+
+data class Layer(val n: String, val m: Bitmap) {
```

Comment:
This isn't part of this ticket yet so there could be lots more added scope than just a name and bitmap, like layer blend mode etc and opacity

Reply:
removed, your right, it isn't in ticket scope.